contents/scalability/scalable-development.md

```diff
@@ -33,9 +33,29 @@ There are a few common bottlenecks that can be avoided such as database querying
 
 There are many program profilers available that you can use to find the bottlenecks in your server. By finding out where the server spends most computational time in, you can potentially save huge amounts of time.
 
-Performance bottlenecks ([resource](https://www.apicasystem.com/blog/5-common-performance-bottlenecks/))
+1. More details on the common performance bottlenecks ([resource](https://www.apicasystem.com/blog/5-common-performance-bottlenecks))
 
-Non-Relational Database ([resource](http://www.jamesserra.com/archive/2015/08/relational-databases-vs-non-relational-databases/)) ([resource](https://www.pluralsight.com/blog/software-development/relational-non-relational-databases))
+1. Non-Relational Database ([resource](http://www.jamesserra.com/archive/2015/08/relational-databases-vs-non-relational-databases)) ([resource](https://www.pluralsight.com/blog/software-development/relational-non-relational-databases))
+
+### Caching
```

Comment:
You may also wish to discuss [sharding](https://stackoverflow.com/questions/992988/what-is-sharding-and-why-is-it-important) since it's more common nowadays and an alternative to caching.

Reply:
Adding a section on sharding is a great idea! However, the section on challenges doesn't fall within my scope just yet. Maybe I'll work on it in another PR! Thanks for the content suggestions :+1: .